src/quo2/components/settings/reorder_item/items/item_tabs.cljs

```diff
@@ -0,0 +1,35 @@
+(ns  quo2.components.settings.reorder-item.items.item-tabs
+  (:require [quo2.components.tabs.segmented-tab :as quo2]
+            [quo.react-native :as rn]
+            [quo.components.text :as text]
+            [quo2.components.settings.reorder-item.style :as style]
+            [quo2.components.icon :as quo2-icons]))
+
+(defn transform-data
+  [data]
+  (map #(assoc %
+               :id (:id %)
+               :label [rn/view
```

Comment:
I would probably extract this as a separate function, for clarity

Reply:
Done